fetch-utils.js

```diff
@@ -4,27 +4,37 @@ const SUPABASE_KEY =
 const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
 export async function getCountries(name, continent) {
-    // > Part D: Add a second argument to `select(` to 
+    // > Part D: Add a second argument to `select(` to
     // return an exact db count of matching records
 
     // > Part A: Implement the client query from countries:
+
     //   1. select all columns
+
     //   2. order by country name
     //   3. limit to 100 countries
-    let query = // ?
+    let query = client.from('countries').select('*', { count: 'exact' }).order('name').limit(100);
+    // ?
 
     if (name) {
         // > Part C: add query for name
+        query = query.ilike('name', `%${name}%`);
```

Comment:
get used to seeing this — taking a request or response, passing it to something which alters the request/response then passes it to something else